packages/core/contracts/UnirepSocial.sol

```diff
@@ -92,6 +101,8 @@ contract UnirepSocial {
         postReputation = _postReputation;
         commentReputation = _commentReputation;
         airdroppedReputation = _airdroppedReputation;
+        isClaimed[_userName] = true;
+        unirep.userNames.push(_userName);
```

Comment:
There is no `userNames` in `unirep`.
Just store the `userNames` in Unirep Social contract.